### PR TITLE
Added some missing information to changelog

### PR DESCRIPTION
**Pull request by kolibril13**
See here what is added:
    #. `FRAME_HEIGHT` to `config["frame_width"]`
    #. `TOP` to `config["frame_height"] / 2 * UP`
    #. `BOTTOM` to `config["frame_height"] / 2 * DOWN`
    #. `LEFT_SIDE` to `config["frame_width"] / 2 * LEFT`
    #. `RIGHT_SIDE` to `config["frame_width"] / 2 * RIGHT`
    #. `self.camera.frame_rate` to `config["frame_rate"]`

Further, a small correction to one example

### CODE REVIEW

**Review by leotrs — APPROVED** [review]

**Comment by kolibril13** [comment]
Wow, the development cycles are super fast currently! :)

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -42,6 +42,15 @@ Config system
 #. Implement a :code:`manim.cfg` config file system, that consolidates the global configuration, the command line argument parsing, and some of the constants defined in :code:`constants.py`
 #. Added utilities for manipulating Manim’s :code:`.cfg` files.
 #. Added a subcommand structure for easier use of utilities managing :code:`.cfg` files
+#. Also some Variables changed to the new config system:
+    #. `FRAME_HEIGHT` to `config["frame_width"]`
+    #. `TOP` to `config["frame_height"] / 2 * UP`
+    #. `BOTTOM` to `config["frame_height"] / 2 * DOWN`
+    #. `LEFT_SIDE` to `config["frame_width"] / 2 * LEFT`
+    #. `RIGHT_SIDE` to `config["frame_width"] / 2 * RIGHT`
+    #. `self.camera.frame_rate` to `config["frame_rate"]`
```
Comment (leotrs) [inline docs/source/changelog.rst:51]:
```suggestion
#. Also some variables have been moved from constants.py to the new config system:
    #. ``FRAME_HEIGHT`` to ``config["frame_width"]``
    #. ``TOP`` to ``config["frame_height"] / 2 * UP``
    #. ``BOTTOM`` to ``config["frame_height"] / 2 * DOWN``
    #. ``LEFT_SIDE`` to ``config["frame_width"] / 2 * LEFT``
    #. ``RIGHT_SIDE`` to ``config["frame_width"] / 2 * RIGHT``
    #. ``self.camera.frame_rate`` to ``config["frame_rate"]``
```

**Review by leotrs — CHANGES_REQUESTED** [review]

File: docs/source/changelog.rst (hunk 1 above)
Comment (leotrs) [reply docs/source/changelog.rst:51]: use double backticks to render code - see https://manimce--603.org.readthedocs.build/en/603/changelog.html#v0-1-0

**Comment by kolibril13** [comment]
Thanks for catching the double backticks

File: docs/source/changelog.rst [hunk 2]
```diff
@@ -42,6 +42,15 @@ Config system
 #. Implement a :code:`manim.cfg` config file system, that consolidates the global configuration, the command line argument parsing, and some of the constants defined in :code:`constants.py`
 #. Added utilities for manipulating Manim’s :code:`.cfg` files.
 #. Added a subcommand structure for easier use of utilities managing :code:`.cfg` files
+#. Also some variables have been moved from constants.py to the new config system:
```
Comment (behackl) [inline docs/source/changelog.rst:45]: I think you don't want this line to be printed bold; it is currently rendered as a definition list. (Although this missing newline is actually missing several times throughout the documentation... if you are at it, I'd appreciate if you could add it for the other nested lists as well. :-)

```suggestion
#. Also some variables have been moved from ``constants.py`` to the new config system:

```

**Review by behackl — CHANGES_REQUESTED** [review]

File: docs/source/changelog.rst (hunk 2 above)
Comment (kolibril13) [reply docs/source/changelog.rst]: Done, also fixed it in the other nested list

**Review by behackl — APPROVED** [review]
Thank you, now this LGTM!

Please remember to push to your fork of the repository, instead of the org repo to reduce the number of branches flying around (#590).

**Review by leotrs — APPROVED** [review]

**Pull request merged**